docs/site/content/docs/latest/tanzu-debugging-tips.md

```diff
@@ -31,27 +31,21 @@ The previous shows a list of active containers in docker (the output for your
 environment will vary). Make note of the container ID as it will be used in the
 following steps.
 
-1. Next, identify the controller-manager pod for your provider (for instance,
-   for AWS it is `capa-controller-manager-*` in the `capa-system` namespace)
-
+1. Next, Retrieve the log for your infrastructure providers controller pod to verify why it is unable to complete
+   the management cluster provisioning:
     ```sh
-    docker exec <CONTAINER_ID> kubectl get po --namespace capa-system --kubeconfig /etc/kubernetes/admin.conf
-    NAMESPACE                           NAME                                                                  READY   STATUS    RESTARTS   AGE
-    capa-system                         capa-controller-manager-<UINQUE_ID>                              2/2     Running   0          19m
-    ```
-
-    The previous command will vary depending on your management cluster provider, as shown below:
-
-    * AWS: `capa-controller-manager` / `capa-system`
-    * Azure: `capz-controller-manager` / `capz-system`
-    * docker: `capd-controller-manager` / `capd-system`
-    * vSphere: `capv-controller-manager` / `capv-system`
-
-1. Retrieve the log for that controller to verify why it is unable to complete
-   the management cluster provision:
-
-    ```shell
-    docker exec <CONTAINDER_ID> kubectl logs capa-controller-manager-<UNIQUE_ID> -n capa-system -c manager --kubeconfig /etc/kubernetes/admin.conf
+    # For AWS
```

Comment:
I don't think the mdlinter will like using top level headers inside the bullet list. You may need to change this to `##` or even `###`. Might want that for formatting purposes too.

Reply:
I've changed it to ##. mdlint didnt complain about a single # but always better to play it safe.